.github/workflows/images-legacy-releases.yaml

```diff
@@ -0,0 +1,100 @@
+name: Image Release Build
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+      - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+    branches:
+      - pr/test-docker-images-legacy
+
+jobs:
+  build-and-push:
+    environment: release
```

Comment:
How does this interact with forks? If we mirror Cilium master somewhere and mirror the `v*` tags, will this automatically try to build an image on each mirror repository of Cilium as well? 

I believe we need an `if: github.repository == 'cilium/cilium'` statement here.

Reply:
oh I must have removed it by accident, that if statement was there in the beginning. Thanks for spotting it.